File: 36-Extra-Challenges/112-Titleize.py
def titleize(string):
    word_list = string.split()
    titled = [word[0].upper() + word[1:] for word in word_list]
    return ' '.join(titled)

    
    
print(titleize('this is awesome'))
print(titleize('oNLy cAPITALIZe fIRSt'))

File: 36-Extra-Challenges/112-Titleize.js
const titleize = (str) => {
    return str.split(' ').map(word => word[0].toUpperCase() + word.slice(1)).join(' ')
}

    
    
console.log(titleize('this is awesome'))
console.log(titleize('oNLy cAPITALIZe fIRSt'))